.github/linters/.yaml-lint.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Just to be clear, this is to lint the one yaml file?

Reply:
Heh, yes,  to "lint" the "linter" configuration.  

Note though in the linter config, the linter used for the codebase is: github/super-linter/slim@v5